06_neighborhoods/Transportation/transit_cost_county.qmd

```diff
@@ -0,0 +1,296 @@
+}
+```
+
+1 missing value: Loving County, TX (48301 FIPS).
+
+County-Level Transportation Cost 2019
+
+```{r}
+ggplot(transportation_cost_county_2019, aes(x=t_80ami)) + geom_histogram(binwidth=10) + labs(y="number of counties", x="Annual Transit Cost for the Regional Moderate Income Household, 2019")
+```
+
+Look at summary stats
+```{r}
+summary(transportation_cost_county_2019$t_80ami) 
+```
+
+Examine outliers
+```{r}
+transportation_cost_county_2019_outliers <- transportation_cost_county_2019 %>% 
+  filter(t_80ami>100) 
+```
+
+No weird outliers
+
+Use stopifnot to check if all values in "transportation_cost_county_2019" are non-negative
+
+```{r}
+stopifnot(min(transportation_cost_county_2019$t_80ami, na.rm = TRUE) >= 0)
+```
+
+Good to go.
+
+Find indices of missing values for the "transit_cost_80ami" variable
+
+```{r}
+missing_indices <- which(is.na(transportation_cost_county_2019$t_80ami))
+```
+
+Print observations with missing values
+
+```{r}
+if (length(missing_indices) > 0) {
+  cat("Observations with missing values for transit_cost_80ami:\n")
+  print(transportation_cost_county_2019[missing_indices, , drop = FALSE])
+} else {
+  cat("No missing values for transportation_cost_county_2019\n")
+}
+```
+
+No missing values for 2019.
+
+## Data quality marker
+
+Determine data quality cutoffs based on number of observations (all at the HH level for these values)
+
+```{r}
+summary(transportation_cost_county_2015$households) 
+summary(transportation_cost_county_2019$households) 
+```
+
+We use a 30 HH cutoff for Data Quality 3 for the ACS variables, so for the sake of consistency, since none of these are less than 30 (all minimum values are at least 30 HHs), Data Quality can be 1 for all these observations BUT ALSO, rename all the metrics variables to what we had before (transit_trips & transit_cost), so we can name the quality variable appropriately
+
+```{r}
+transportation_cost_county_2015 <- transportation_cost_county_2015 %>% 
+  rename(transit_cost = t_80ami) %>%
+  mutate(transit_cost_quality = 1)
+transportation_cost_county_2019 <- transportation_cost_county_2019 %>% 
+  rename(transit_cost = t_80ami) %>%
+  mutate(transit_cost_quality = 1)
+```
+
+## Export files
+
+Add a variable for the year of the data
+
+```{r}
+transportation_cost_county_2015 <- transportation_cost_county_2015 %>%
+  mutate(
+    year = 2015
+  )
+```
+
+```{r}
+transportation_cost_county_2019 <- transportation_cost_county_2019 %>%
+  mutate(
+    year = 2019
+  )
+```
+
+Combine the two years into one overall files for both variables
+
+```{r}
+transit_cost_county <- rbind(transportation_cost_county_2015, transportation_cost_county_2019)
+```
+
+Keep variables of interest and order them appropriately also rename to correct var names
+
+```{r}
+transit_cost_county <- transit_cost_county %>%
+  rename(index_transportation_cost = transit_cost,
+         index_transportation_cost_quality = transit_cost_quality) %>%
```

Comment:
I don't understand using the wrong name throughout the script and then renaming at the end. It's unclear.